core/templates/dev/head/library/library.html

```diff
@@ -59,10 +59,15 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
           <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index">
             <div class="row oppia-library-group-header-container">
               <h2 class="oppia-library-group-header">
-                <a ng-click="showAllResultsForCategories(group.categories)" translate="<[group.translationId]>">
+                <a ng-if="libraryGroups[$index].header != 'Featured Activities'"
```

Comment:
FYI, it will, I think -- we discussed this on gitter. Turns out that the 'header' key in the JS is the English version of the string that then gets converted into the i18n id.

(@Arunabh98, you checked, right?)

Though, it's true that this has caused confusion several times. I wonder whether it's better to send, from the backend, the I18N key instead.